### PR TITLE
Add cmd/crier to awesomeify GitHub comments.

### PR DESCRIPTION
**Pull request by spxtr**
Sorry, this is a huge PR. It doesn't make sense to make multiple PRs, unfortunately. I'll probably add a few more tests tomorrow, but the most important things are covered.

### CODE REVIEW

**Comment by spxtr** [comment]
ping @rmmh @apelisse

File: prow/crier/crier.go [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+
+func NewServer(ghc GitHubClient) *Server {
+	return &Server{
+		rc:  make(chan Report),
+		ghc: ghc,
+	}
+}
+
+func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	if r.Method != http.MethodPost {
```
Comment (rmmh) [inline prow/crier/crier.go]: You should restrict the URL here too, so it's easier to extend this in the future.

Comment (spxtr) [inline prow/crier/crier.go]: Done.

**Pull request merged**